resources/views/restaurant/adding.blade.php

```diff
@@ -94,12 +94,37 @@
             </div>
             
             <div class="mb-3">
-                <label for="photo" class="form-label h3">Photo</label>
-                <div class="py-5 bg-secondary" id="photo">
-                <input type="file" name="photo" multiple>
-                    Photo uploading area
+                <label for="photo" class="form-label h3">Photo</label><br>
+                <div class="row">
+                    <div class="col-4">
+                        <input type="file" name="photo">
+                    </div>
+                    <div class="col-4">
+                        <input type="file" name="photo">
+                    </div>
+                    <div class="col-4">
+                        <input type="file" name="photo">
+                    </div>
                 </div>
 
+        <!--photo uploading design-->
+            <!-- <div>
+                <label for="inputTag">
+                    Select Image <br/>
+                    <i class="fa fa-2x fa-camera"></i>
+                    <input id="inputTag" type="file"/>
+                    <br/>
+                    <span id="imageName"></span>
+                </label>
+                </div>
+                <script>
+                    let input = document.getElementById("inputTag");
+                    let imageName = document.getElementById("imageName")
+                    input.addEventListener("change", ()=>{
+                        let inputImage = document.querySelector("input[type=file]").files[0];
+                        imageName.innerText = inputImage.name;
+                    })
```

Comment:
Same as well in this section. We can remove the comments if they're not being used.